router-protobuf/src/main/kotlin/io/moia/router/proto/ProtoDeserializationHandler.kt

```diff
@@ -1,20 +1,26 @@
 package io.moia.router.proto
 
 import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent
-import io.moia.router.DeserializationHandler
-import io.moia.router.contentType
 import com.google.common.net.MediaType
 import com.google.protobuf.Parser
+import io.moia.router.DeserializationHandler
+import io.moia.router.contentType
+import isCompatibleWith
 import java.util.Base64
 import kotlin.reflect.KClass
 import kotlin.reflect.KType
 import kotlin.reflect.full.staticFunctions
 
 class ProtoDeserializationHandler : DeserializationHandler {
     private val proto = MediaType.parse("application/x-protobuf")
+    private val protoStructuredSuffixWildcard = MediaType.parse("application/*+x-protobuf")
 
     override fun supports(input: APIGatewayProxyRequestEvent): Boolean =
-        input.contentType() != null && MediaType.parse(input.contentType()).`is`(proto)
+        if (input.contentType() == null)
+            false
```

Comment:
I believe you are missing this test case. I was going to write it for you but I'm having that weird kotlin `overload` problem